lib/core/src/Cardano/Pool/DB/Sqlite/TH.hs

```diff
@@ -44,12 +44,13 @@ share
 -- The set of stake pools that produced a given block
 PoolProduction sql=pool_production
     poolProductionPoolId         W.PoolId     sql=pool_id
-    poolProductionSlot           W.SlotId     sql=slot
+    poolProductionSlot           Word16       sql=slot
+    poolProductionEpoch          Word64       sql=epoch
```

Comment:
Why here? We have to be careful with this.

Reply:
👍 It turns out this is in fact not needed. Dropped.